### PR TITLE
Adding junifer

### PR DESCRIPTION
**Pull request by synchon**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by synchon** [comment]
Hi, I'm not sure if I should add `noarch: generic` or `noarch: python` as I have a constraint of Python >= 3.8, @conda-forge-admin, please ping conda-forge/help-python .

**Comment by synchon** [comment]
This is my confirmation for willing to be listed as the recipe maintainer.

**Comment by synchon** [comment]
@conda-forge-admin, please ping team

File: recipes/junifer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/junifer-{{ version }}.tar.gz
+  sha256: 9244c954433e97858868b09d06adfdf20d238d96f1f7f5c276d712ab69e438a2
+
+build:
+  skip: true  # [py<38]
```
Comment (dopplershift) [inline recipes/junifer/meta.yaml:13]: Better to make this noarch:
```suggestion
  noarch: python
```

File: recipes/junifer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - python >=3.8
+    - click >=8.1.3,<8.2
+    - numpy >=1.22,<1.24
+    - datalad >=0.15.4,<0.19
+    - pandas >=1.4.0,<1.6
+    - nibabel >=3.2.0,<4.1
+    - nilearn >=0.9.0,<=0.10.0
+    - sqlalchemy >=1.4.27,<=1.5.0
+    - pyyaml >=5.1.2,<7.0
+    - importlib-metadata  # [py<310]
```
Comment (dopplershift) [inline recipes/junifer/meta.yaml:36]: You can't use selectors in noarch recipes. In this case, it's better to just unconditionally depend on `importlib-metadata`:
```suggestion
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor tweaks and this is good to go.

**Comment by synchon** [comment]
@dopplershift Thanks for the initial review! I believe I have addressed your comments. :)

**Comment by synchon** [comment]
@conda-forge-admin, please ping team

**Review by dopplershift — APPROVED** [review]

**Pull request merged**